products/devsuite/docs-and-apis.adoc

```diff
@@ -0,0 +1,3 @@
+:awestruct-layout: product-docs-and-apis
+:awestruct-interpolate: true
```

Comment:
Maybe there's more coming, but if there's no content, do we need the interpolate?